articles/storage/common/storage-use-azcopy-files.md

```diff
@@ -57,7 +57,7 @@ This section contains the following examples:
 > For a complete list, see [options](storage-ref-azcopy-copy.md#options).
 
 > [!NOTE]
-> AzCopy doesn't automatically calculate and store the file's md5 hash code. If you want AzCopy to do that, then append the `--put-md5` flag to each copy command. That way, when the file is downloaded, AzCopy calculates an MD5 hash for downloaded data and verifies that the MD5 hash stored in the file's `Content-md5` property matches the calculated hash.
+> AzCopy doesn't automatically calculate and store the file's md5 hash code .If you want AzCopy to do that, then append the `--put-md5` flag to each copy command. That way, when the file is downloaded, AzCopy calculates an MD5 hash for downloaded data and verifies that the MD5 hash stored in the file's `Content-md5` property matches the calculated hash.For smaller blobs (<256MB IIRC), the service computes md5 hash code automatically.
```

Comment:
Hi @SurbhiSingh1712 - thanks for the contribution! I have a question about this proposed change. The line you're adding at the end here appears to contradict the rest of this note. The note states that the MD5 hash code is no automatically calculated. Your addition at the end states that it is (albeit in small files). Am I reading this proposed change correctly? Also, on a smaller note, it appears that you accidentally added a space at the end of the first sentence of this note.